### ISSUE TITLE
Ubuntu fix-permissions.js failed

### ISSUE THREAD

**Issue by AngeIII** [DevExpress/testcafe#2144]
Error in javascript possibly `-R` on ubuntu should be last in string
### What is the current behavior?
Error: Command failed: chmod -R +x ./bin/linux/*

### What is the expected behavior?
File permissions fixed

### How would you reproduce the current behavior (if this is a bug)?
sudo npm install -g testcafe

### Should be fixed by changing args:
```
var binariesPath = '';

if (platform === 'darwin')
    binariesPath = '-R ./bin/mac/*';
else if (platform === 'linux')
    binariesPath = './bin/linux/* -R';

if (!binariesPath)
    return;

exec('chmod +x ' + binariesPath, function (err) {
    if (err)
        process.stdout.write(err.stack + '\n');
    else
        process.stdout.write('File permissions fixed\n');
});

```

### Specify your

* operating system: Ubuntu
* testcafe version: 0.18.6
* node.js version:

**Comment by AndreyBelym** [comment]
Hi @AngeIII, it must be fixed in `0.190-alpha1`, could you please try it with `npm i testcafe@0.19.0-alpha1`?

**Comment by AngeIII** [comment]
@AndreyBelym looks like this file is not exists anymore in 0.19, so probably fixed. 
npm install works fine.
